src/libs/lib.c

```diff
@@ -573,25 +573,47 @@ static void dt_lib_presets_popup_menu_show(dt_lib_module_info_t *minfo)
   darktable.gui->presets_popup_menu = GTK_MENU(gtk_menu_new());
   menu = darktable.gui->presets_popup_menu;
 
+  const gboolean hide_default = dt_conf_get_bool("plugins/lighttable/hide_default_presets");
+  const gboolean default_first = dt_conf_get_bool("plugins/darkroom/default_presets_first");
```

Comment:
this is not anymore for darkroom only, worth renaming it to `module/default_presets_first` or just `default_presets_first`.

Reply:
Yes, that would be better, but that will mean that older configs will be lost too.
That said, it has been added recently, so we can maybe consider that this is not a problem...

Reply:
This is super recent config, for maybe over a month in master :) anybody who changed it will have no problem changing it back imo ;)

Reply:
Yes, not a problem people using master are aware of such possible issue. So changing is not a problem to me.

Reply:
done